src/wallet/salvage.cpp

```diff
@@ -27,6 +28,7 @@ bool RecoverDatabaseFile(const fs::path& file_path, bilingual_str& error, std::v
     DatabaseStatus status;
     options.require_existing = true;
     options.verify = false;
+    options.require_format = DatabaseFormat::BERKELEY;
```

Comment:
Is this another bugfix we should put in 0.21?

Reply:
It's not strictly necessary. Other stuff later like trying to actually open the wallet file if it isn't BDB will still fail and we won't do anything to the file. It will just have a slightly worse error message.